doc/source/examples/multiphysics/sloshing-in-rectangular-tank/sloshing-in-rectangular-tank.rst

```diff
@@ -4,16 +4,18 @@ Sloshing in a Rectangular Tank
 
 This example simulates the damping of a small amplitude wave for Reynolds number of (:math:`2`, :math:`20`, :math:`200` and :math:`2000`). The problem is inspired by the test case of Carrica *et al.* [#carrica2007]_
 
+****
 
 --------
 Features
 --------
 
 - Solver: ``lethe-fluid`` 
 - Volume of fluid (VOF)
-- Unsteady problem handled by an adaptive BDF2 time-stepping scheme 
+- Unsteady problem handled by an adaptive BDF2 time stepping scheme
```

Comment:
```suggestion
- Unsteady problem handled by an adaptive BDF2 time-stepping scheme
```
time-stepping is a compound noun and takes an hyphen in this case. I am pretty sure of that.